### PR TITLE
README Overhaul

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Major README overhaul  Bug in GitHub preventing preview, so hopefully everything is formatted correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,63 +30,75 @@ Index
...
+------------
+Download the latest release from the [Google Play Store](https://play.google.com/store/apps/details?id=dev.ukanth.ufirewall), [GitHub](https://github.com/ukanth/afwall/releases), or [F-Droid](https://f-droid.org/repository/browse/?fdid=dev.ukanth.ufirewall)
+
+The [changelog](https://github.com/ukanth/afwall/blob/beta/Changelog.md) documents changes between each release.
+
+Supports
 --------
-* Support 4.x/5.x and 6.x versions of Android
-* Easy to install & simple to use
+* Android versions 4.x/5.x/6.x
```
Comment (ukanth) [inline README.md]: Please add Nougat as well @Gitoffthelawn

File: README.md [hunk 2]
```diff
@@ -28,63 +30,75 @@ Index
...
-* The firewall cannot prevent corporate data from being copied to a memory stick or HDD and these are removed from the building. 
-* AFWall+ does not scan for virus/malware that may exist in the files that pass through it, because a firewall is not an antivirus solution.
+* A firewall cannot prevent corporate data from being copied to a memory stick or HDD, and having these removed from the building. 
+* AFWall+ does not scan for virus/malware that may exist in the files that pass through it, because it is a firewall and not an antivirus solution.
 
 Compatibility
 -------------
 
-AFWall+ has been success tested with Android version 4.xx - 5. (ICS, JellyBean, KitKat, Lollipop) and is reported to work with most Android variants, including stock ROMs.
+AFWall+ has been successfully tested with Android version 4.xx - 6. (ICS, JellyBean, KitKat, Lollipop, Nougat) and is reported to work with most Android variants, including stock ROMs.
```
Comment (ukanth) [inline README.md]: Same as above comment, instead of 6, it should be 7

File: README.md [hunk 3]
```diff
@@ -28,63 +30,75 @@ Index
...
+We do not recommend using AFWall+ in combination with any of the similar solutions because this could result in conflicts and potential data leaks (iptables could get overwritten).
 
 Upgrading
 ---------
 
-* **Make a backup** (e.g. with Titanium Backup)
-* **Do not remove the previous version** (or else your settings might get lost)
-* Download the new version
-* Install the new version over the previous version
+* **Make a backup** (e.g. make a Nandroid backup or use Titanium Backup).
```
Comment (ukanth) [inline README.md]: Nandroid backup is not required , Because it doesn't touch anything at system level files @Gitoffthelawn

**Comment by Gitoffthelawn** [comment]
@ukanth GitHub is reporting activity on this from 6 hours ago, but not specifying the activity.  Did you perform an action?

**Comment by ukanth** [comment]
Yes, There are couple of review comments I tagged you.

**Comment by Gitoffthelawn** [comment]
@ukanth Weird. It's not showing me any review comments.  If you can, post a link.

**Comment by ukanth** [comment]
@Gitoffthelawn  https://github.com/ukanth/afwall/pull/626/files ( check show comments ) 
You should be able to see the comments

**Review by Gitoffthelawn — COMMENTED** [review]
@ukanth Adding a test comment.

**Comment by Gitoffthelawn** [comment]
@ukanth I believe you, and I apologize for asking, but I see no `show comments` checkbox.  Can you post a screenshot so I can compare what you see to what I see?  Perhaps because you are the repo owner it shows it differently to you.

**Comment by Gitoffthelawn** [comment]
@ukanth Thank you for your review!  All requested changes implemented.

**Pull request merged**